src/data/calendar.js

```diff
@@ -0,0 +1,136 @@
+/* global getBoxingDays, getFrenchBankHolidays */
+(function(exports) {
+  "use strict";
+
+  function guessTypeFromComment(comment) {
+    var KNOWN_TYPES = ['RTT', 'CP', 'CS'];
```

Comment:
My point is that it's really easy to be prevented by using regexps and `\b` ("edge of a word"). But this can be for a future patch, we don't need this perfect now :)